packages/api/middlewares/checkAccess.js

```diff
@@ -3,6 +3,7 @@ const get = require("lodash/get");
 
 module.exports = () => {
   return async (req, res, next) => {
+    return next();
```

Comment:
Please correct me If I am wrong when you use return next(), it will jump out the callback immediately and the code below return next() in the callback will be unreachable. is not it ? @soumyadip007

Reply:
resolving

Reply:
Yes that is added by mistake, resolved

Reply:
If that block is not in use, then completely remove it. If required, then you can always pull it from history @soumyadip007